<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>780</width>
      <height>570</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>780</width>
      <height>570</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	background-color: rgb(255, 255, 255);
}
QFrame#frame{
	
	border-image: url(:/mainbgp.jpg);
	border-radius:20px
}
#frame_2{
	background-color: rgba(102, 102, 102, 100);
	border-radius:20px
}
#frame_3{
	background-color: rgba(93, 93, 93, 150);
	border-radius:20px
}
QLabel#welcomelb{
	background-color: rgba(255, 255, 255, 0);
}
#ratelb{
	background:transparent;
}
#timelb{
	background:transparent;
}
#headlb{
	background:transparent;
}
#greetlb{
	background:transparent;	
}
QLineEdit#lineEdit{
	
	background:transparent;
	border:none;
	border-bottom:1px solid rgb(255, 255, 255);
}
#downloadPathEdit{
	
	background:transparent;
	border:none;
	border-bottom:1px solid rgb(255, 255, 255);
}
QPushButton#closebtn{
	background: transparent;
}
#minbtn{
	background: transparent;
}
#browseButton{
		background: rgba(0, 136, 199, 150)
}
#resetPathButton{
		background: rgba(0, 136, 199, 150)
}
#pushButton{
		background: rgba(0, 136, 199, 150)
}
#stopbtn{
	background: rgba(0, 136, 199, 150)
}#continuebtn{
	background:rgba(0, 136, 199, 150)
}#cancelbtn{
	background:rgba(0, 136, 199, 150)
}
#helpbtn{
	background:transparent;
	font: italic 8pt &quot;Arial&quot;;
	text-decoration: underline;
}
#historybtn{
	background:transparent;
	font: italic 8pt &quot;Arial&quot;;
	text-decoration: underline;
}
QChartView#graphicsView{
	background:transparent;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>780</width>
       <height>570</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>780</width>
       <height>570</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="welcomelb">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; color:#ffffff;&quot;&gt;Welcome&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>670</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Please input the url of the files you wanna download</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="downloadPathEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Download path (leave empty for default)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>240</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetPathButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>240</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>400</y>
        <width>670</width>
        <height>30</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="ratelb">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>440</y>
        <width>69</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="timelb">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>440</y>
        <width>69</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>780</width>
        <height>40</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="closebtn">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>5</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="pics.qrc">
         <normaloff>:/closepic.png</normaloff>:/closepic.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>33</width>
         <height>33</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="minbtn">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>5</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="pics.qrc">
         <normaloff>:/minpic.png</normaloff>:/minpic.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>33</width>
         <height>33</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="headlb">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>5</y>
         <width>220</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;&gt;MyDownloader&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>670</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLabel" name="greetlb">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="stopbtn">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>341</y>
        <width>211</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="continuebtn">
      <property name="geometry">
       <rect>
        <x>266</x>
        <y>341</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelbtn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>340</y>
        <width>221</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="helpbtn">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>480</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Any Question?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="historybtn">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>480</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>History</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
